ui/prometheus-plugin/src/plugins/PrometheusDatasourceEditor/PrometheusDatasourceEditor.tsx

```diff
@@ -64,6 +64,19 @@ export function PrometheusDatasourceEditor(props: PrometheusDatasourceEditorProp
             }}
             onChange={(e) => onChange({ ...value, direct_url: e.target.value })}
           />
+          <Typography py={2} variant="h4">
+            Prometheus Settings
+          </Typography>
+          <TextField
+            fullWidth
+            label="Scrape Interval"
+            value={value.scrape_interval || ''}
+            placeholder="Default: 15s"
```

Comment:
I'm wondering if we should use Prometheus default scrape interval which is 1m according to their documentation.
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config:~:text=%5B%20scrape_interval%3A%20%3Cduration%3E%20%7C%20default%20%3D%201m%20%5D

Reply:
I used the same default scrape interval of Grafana. I didn't check Prometheus, it makes sense to use the same as Prometheus